eLearning-courseware/eLearning.jsonld

```diff
@@ -0,0 +1,1271 @@
+{
+   "@context": "https://w3id.org/xapi/profiles/context",
+    "id": "https://w3id.org/xapi/eLearning",
+    "type": "Profile",
+    "conformsTo": "https://w3id.org/xapi/profiles#1.0",
+    "prefLabel": {
+        "en": "xAPI eLearning Courseware Profile"
+    },
+    "definition": {
+        "en": "The xAPI eLearning Courseware Profile describes a set of interactions used for level three interactive distance learning."
+    },
+    "seeAlso": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile",
+    "versions": [
+        {
+            "id": "https://w3id.org/xapi/eLearning/v1.0",
+            "generatedAtTime": "2017-10-09T20:04:39.214Z"
+        }
+    ],
+    "author": {
+        "type": "Organization",
+        "name": "Riptide Elements"
+    },
+    "concepts": [
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/concepts/course",
+        "inScheme": "https://w3id.org/xapi/eLearning",
```

Comment:
This should be the current version; membership in the overall profile comes via the "concepts" relationship

Reply:
(a general comment for all the uses of inScheme)

Reply:
Can you clarify what you mean here? Re-using the top-level `id` field seems to be how every profile and the [template](https://github.com/RiptideElements/xapi-authored-profiles/blob/master/starter-template.jsonld#L26) use this `inScheme` field.

Reply:
I haven't reviewed all of them, but you can see how it works in cmi5, which I've spent more time helping out with: https://github.com/adlnet/xapi-authored-profiles/blob/master/cmi5/cmi5.jsonld

The overall profile ID is `https://w3id.org/xapi/cmi5`, the current version is `https://w3id.org/xapi/cmi5/context/categories/cmi5`, and all concepts and so forth are `"inScheme": "https://w3id.org/xapi/cmi5/context/categories/cmi5"`

Reply:
Essentially, concepts is an inverse-sub-relationship of inScheme, so inScheme is not needed again for the overall profile. What's needed is a connection to the specific current version (this is all following recommended best practices for SKOS versioning), so that it is possible to answer questions via SPARQL query such as "what concepts are present in this version that aren't present in the previous version?"

Reply:
I'll file a defect for the other profiles

Reply:
This makes sense after you explained it. Good explanation, thanks. I have updated all the `inScheme` properties to `"inScheme": "https://w3id.org/xapi/eLearning/v1.0"` as suggested.